lesson11/index.js

```diff
@@ -0,0 +1,66 @@
+const ul = document.getElementById('js-lists');
+
+const fetchData = async() => {
+    const response = await fetch('https://jsondata.okiba.me/v1/json/KN8s0211027113740');
+    const obj = await response.json();
+    const listData = obj.data;
+
+    displayLoadingImage ();
+
+    await new Promise(resolve => {
+        setTimeout(() => {
+            resolve(listData);
+        }, 3000);
+    });
+    createList(listData);
+}
+
+const getData = async() => {
+    try {
+        return await fetchData();
+    } catch (error) {
+        errorText("エラーが発生しました");
+        return error;
+    } finally {
+        removeLoadingImage ();
+    }
+}
```

Comment:
Let's run createList() in this part.

Reply:
IMO
I agree with what Mr. Kimura is telling us. Let me add to that.

```js
const getData = async() => {
   displayLoadingImage ();
    try {
        return await fetchData();
    } catch (error) {
        errorText("エラーが発生しました");
        return error;
    } finally {
        removeLoadingImage ();
    }
}
```

Because the job of fetching data has nothing to do with the DOM.

Reply:
I changed the function name and put the function displayLoadingImage.
I fixed -> e5c45d0